export const sendMessage = async (req, res) => {
  try {
  } catch (error) {
    console.log("Error in sendMessage controller: ", error.message);
  }
};
